frontend/packages/dev-console/src/components/topology/topology-types.ts

```diff
@@ -151,3 +152,13 @@ export type NodeProvider = (string) => ComponentType<NodeProps>;
 export type EdgeProvider = (string) => ComponentType<EdgeProps>;
 
 export type GroupProvider = (string) => ComponentType<GroupProps>;
+
+export enum KINDS {
```

Comment:
@sahil143 can we be more specific with the name? It can be `ResourceKinds`

Reply:
fixed in https://github.com/openshift/console/pull/2162/commits/7fe748eb664dd4d26deff8e8be7ca583fc832b35